CONTRIBUTING.md

```diff
@@ -8,6 +8,12 @@ We encourage you to read this project's CONTRIBUTING policy (you are here), its
 
 If you have any questions or want to read more, check out the [18F Open Source Policy GitHub repository](https://github.com/18f/open-source-policy), or just [shoot us an email](mailto:18f@gsa.gov).
 
+## Releasing a new version
+
+1. Update the gem version in [`uswds-jekyll.gemspec`](https://github.com/18F/uswds-jekyll/blob/master/uswds-jekyll.gemspec)
+1. `bundle install`
```

Comment:
I think you might need to commit the changes after this step.

Reply:
Would you still need to if we removed the `Gemfile.lock` from this repo? From our convo yesterday, it sounded like you think we should remove that file since it isn't common for published gems to also have a lockfile.

Reply:
If we go with https://github.com/18F/guides-template/issues/84#issuecomment-341022399, we'd want to leave/update the `Gemfile.lock` in the repository. You'd need to commit the change to the gemspec regardless, though.

Reply:
I like the compromise option mentioned in the comment